cmd/argo/commands/delete.go

```diff
@@ -50,6 +50,12 @@ func NewDeleteCommand() *cobra.Command {
 				errors.CheckError(err)
 				workflows = append(workflows, listed...)
 			}
+
+			if len(workflows) == 0 {
+				fmt.Printf("No resources found\n")
+				return
+			}
```

Comment:
We should add this to other commands such as `lint`, `stop`, `resume`, etc. or check that it's already there. Can you do this for us?

Reply:
and we should add tests

Reply:
ok, The “listWorkflows” method is called in the code and need to check if workflows is empty，I find the “list” and “delete” commands have the calls.

Reply:
What does kubectl do when this happens? We should try to be consistent please.

Reply:
Kubectl is the same output:
```
$ kubectl delete pod -l name=myLabel
No resources found

$ kubectl get pod -n test
No resources found in test namespace.
```